### ISSUE TITLE
metrics: time measurement should not be measuring `rm`

### ISSUE THREAD

**Issue by grahamwhaley** [clearcontainers/tests#316]
Presently our 'docker workload time' measurement includes the `-rm` option to docker run, which means we are also measuring the time it takes docker to remove the container image when it has finished. afaik, that removal process does not involve the runtime at all ( @jodh-intel are you able to confirm/deny that?), in which case we are just adding a constant time to our total time we are measuring, which will only help mask actual variations in runtime time.
Thus, most likely remove the `-rm` option from the test, and then add in an extra command to remove the containers after they have finished.

**Comment by grahamwhaley** [comment]
https://github.com/clearcontainers/tests/blob/master/metrics/time/docker_workload_time.sh#L50

**Comment by jodh-intel** [comment]
Yep. `docker run --rm ...` doesn't change the way (or the number of times) the runtime is called so it could be dropped from the test itself. Although, I guess the container(s) should be cleaned up at some point to avoid any potential `ENOSPC` scenarios.
